pkgs/applications/misc/jetbrains-toolbox/default.nix

```diff
@@ -0,0 +1,87 @@
+{ stdenv
+, lib
+, glibc
+, xcbutilkeysyms
+, gcc
+, zlib
+, libsecret
+, xorg
+, libcef
+, libdbusmenu-gtk3
+, libdbusmenu
+, pcre
+, jetbrains
+, fetchzip
+, makeDesktopItem
+, makeWrapper
+, runCommand
+, appimageTools
+, patchelf
+}:
+let
+  pname = "jetbrains-toolbox";
+  version = "1.27.3.14493";
+  name = "jetbrains-toolbox";
+  src = fetchzip {
+    url = "https://download.jetbrains.com/toolbox/jetbrains-toolbox-${version}.tar.gz";
+    sha256 = "sha256-aK5T95Yg8Us8vkznWlDHnPiPAKiUtlU0Eswl9rD01VY=";
+    stripRoot = false;
+  };
+  libPath = lib.makeLibraryPath [ stdenv.cc.cc.lib glibc xorg.xcbutilkeysyms gcc zlib libsecret xorg.libXext ];
+  extractApp = { name, src }: runCommand "${name}-extracted"
+    {
+      buildInputs = [ appimageTools.appimage-exec patchelf gcc libcef ];
+    } ''
+    appimage-exec.sh -x $out ${src}/${name}-${version}/${name}
+    patchelf \
+      --set-rpath "${libPath}:$out/jre/lib:/usr/lib64" \
+      --set-interpreter ${glibc}/lib64/ld-linux-x86-64.so.2 \
+      $out/jetbrains-toolbox
+    patchelf --set-interpreter ${glibc}/lib64/ld-linux-x86-64.so.2 $out/glibcversion
```

Comment:
please see how that is usually done
https://github.com/NixOS/nixpkgs/blob/0af84dc3af38e385fb28ebe16e572087a374e0e7/pkgs/applications/audio/bitwig-studio/bitwig-studio3.nix#L56

Reply:
Fixed.